lib/config/validation.ts

```diff
@@ -522,6 +522,19 @@ export async function validateConfig(
                 }
               }
             }
+            if (key === 'baseBranches') {
+              for (const baseBranch of val as string[]) {
```

Comment:
is it validated to be string array before coming get? 🤔

Reply:
The current schema also allows `[1, 2]`. I've added `subType: 'string'` - not related to this PR but it looks to me as if this was simply missing so far.